_config.yml

```diff
@@ -1,29 +1,18 @@
 permalink: pretty
 markdown: kramdown
-title: Two Factor Auth
-author: Josh Davis
-description:
-  List of sites with Two Factor Auth support which includes SMS, email,
-  phone calls, hardware, and software.
-url: https://2fa.directory
-repository: 2factorauth/twofactorauth
-img: /img/logo.png
-env: development
 twitter: 2faorg
 social_image: https://repository-images.githubusercontent.com/17724730/63495880-b408-11e9-8cc4-03451aa4cc14
-theme_color: "#39CCCC"
```

Comment:
There is still a reference to this variable in [`manifest.json`](https://github.com/2factorauth/twofactorauth/blob/4.0-dev/manifest.json)

https://github.com/2factorauth/twofactorauth/blob/746d73bafac3db0b8e563e7c22be10ee38458c46/manifest.json#L9

Results in `theme_color: "",` at [2fa.carlgo11.com/manifest.json](https://2fa.carlgo11.com/manifest.json)